<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContentWidgetSettings</class>
 <widget class="QWidget" name="ContentWidgetSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string> База Данных</string>
     </property>
     <widget class="QPushButton" name="btnCreateDatabase">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>70</y>
        <width>111</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Создать таблицы</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnDropDatabase">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>100</y>
        <width>111</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Удалить таблицы</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>135</width>
        <height>181</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLineEdit" name="host">
         <property name="placeholderText">
          <string>Хост</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="port">
         <property name="placeholderText">
          <string>Порт</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="database">
         <property name="placeholderText">
          <string>База</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="user">
         <property name="placeholderText">
          <string>Пользователь</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="password">
         <property name="placeholderText">
          <string>Пароль</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="connect">
         <property name="text">
          <string>Подключиться</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_2">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>30</y>
        <width>118</width>
        <height>15</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Состояние:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="state">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="btnRecalcWareHouse">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>130</y>
        <width>111</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Пересчитать склад</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
